src/reanimated2/Hooks.js

```diff
@@ -32,7 +40,7 @@ export function useMapper(fun, inputs = [], outputs = [], dependencies = []) {
 
 export function useEvent(handler, eventNames = [], rebuild = false) {
   const initRef = useRef(null);
-  if (initRef.current === null) {
+  if (initRef.current === null || (Platform.OS === 'web' && rebuild)) {
```

Comment:
Don't we want to rebuild on iOS and Android?

Reply:
it does rebuild on mobile two lines below.